src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+    private final int number;
+    public LottoNumber(int number) throws IllegalArgumentException {
+        this.number = number;
+        this.validateNumber(this.number);
+    }
+    public int getNumber() {
+        return number;
+    }
```

Comment:
```suggestion
    private final int number;
    
    public LottoNumber(int number) throws IllegalArgumentException {
        this.number = number;
        this.validateNumber(this.number);
    }
    
    public int getNumber() {
        return number;
    }
```
이런곳들도 개행해주면 가독성이 좀 더 좋을 것 같다는 생각이 듭니다!

Reply:
필드, 생성자, getter(및 setter)를 한곳에 몰아 두고 싶어서 좀 옹기종기 두었었는데 확실히 개행 하는게 좀더 보기 편한 것 같습니다. 👀 

자바 코드 컨벤션에서도 메서드 선언이 끝난 후 다음 메서드 선언 사이에는 빈줄을 두라고 되어있었네요. 💦